Gemfile.lock

```diff
@@ -65,9 +65,11 @@ GEM
     sass-listen (4.0.0)
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
+    webrick (1.8.1)
 
 PLATFORMS
   ruby
+  x64-mingw-ucrt
```

Comment:
This PR looks good except for I'm not sure what this would be useful for? Is this for something on Windows?